### PR TITLE
Changed navbar and made it more appealing (#90)

### LINKED ISSUE #90: A smaller navbar
#### Description
The navbar display for pc monitors seems a bit big.

#### Expected Behavior
I'd like to tone it down a tad and also add some other hover features to the items.

### PR DESCRIPTION
**Pull request by SushantPatial**
PR fixes #90

### CODE REVIEW

**Comment by san-coding** [comment]
@sudhay23 CI not working here

**Comment by san-coding** [comment]
@SushantPatial 
You have made a lot of changes which undo other contributions , please share a video/screenshots of your changes until the preview build CI checks are working

**Comment by san-coding** [comment]
Solve the readme merge conflicts by deleting from line 1 to line 8
image

**Comment by SushantPatial** [comment]
https://user-images.githubusercontent.com/84243683/135989587-dde8376d-7f57-4086-9e03-88ed83470215.mp4

File: login.html [hunk 1]
```diff
@@ -61,21 +64,6 @@ <h2 class="text-gray-900 text-lg font-medium title-font mb-5">
                     <button type="submit" class="text-white bg-indigo-500 border-0 py-2 px-8 focus:outline-none hover:bg-indigo-600 rounded text-lg">
                         Login
                     </button>
-                    <br>
```
Comment (san-coding) [inline login.html:64]: why have you removed the social login

File: login.html [hunk 2]
```diff
@@ -3,19 +3,22 @@
     <head>
         <title>Login | ShopOnline</title>
         <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```
Comment (san-coding) [inline login.html:6]: why have you remove font awesome

File: README.md [hunk 3]
```diff
@@ -7,8 +7,6 @@
 # ShopOnline - WebApp 
 (Celebrating open-source 🎉)
 
--   We're hosted at [here](https://shoponline-web.000webhostapp.com/) (We are currently facing some issues with registration/login , will be fixed soon)
```
Comment (san-coding) [inline README.md:10]: Why have you removed this line from readme

File: 404.html [hunk 4]
```diff
@@ -48,4 +48,4 @@ <h2>Page not found</h2>
 
 </body>
 
-</html>
```
Comment (san-coding) [inline 404.html:51]: please don't add unnecessary spaces

**Review by san-coding — CHANGES_REQUESTED** [review]

**Comment by san-coding** [comment]
@sudhay23 please review this

**Comment by san-coding** [comment]
@SushantPatial we are working on the CI/CD checks, will merge this soon

**Review by sudhay23 — CHANGES_REQUESTED** [review]
Lets wait for a preview

**Comment by san-coding** [comment]
This PR is behind a few commits and you have made changes to files which are irrelevant to the issue, @SushantPatial please reopen the PR with only the necessary changes, just make the nav bar smaller, don't make any other changes to any other files, you can open a separate issue to add hover effects to the nav elements and I will assign it to you, please work on a specific issue so that we can keep the pull requests clean and to the point , thank you , looking forward to merging you PR, please fetch and  merge  before you commit your changes in the next PR (just reduce nav bar length) 👍🏻

**Comment by SushantPatial** [comment]
Okay @san-coding I'll work on it and make a new PR

**Comment by san-coding** [comment]
> Okay @san-coding I'll work on it and make a new PR

Yes, make sure you stick to the issue and don't change any other files, looking forward to merging your PR

**Pull request closed without merging**